medical/models/medical_patient.py

```diff
@@ -60,110 +35,88 @@ class MedicalPatient(orm.Model):
     form around medical.
     '''
     _name = 'medical.patient'
+    _description = 'Medical Patient'
     _inherits = {'res.partner': 'partner_id', }
 
-    def _get_default_patient_id(self, cr, uid, context=None):
+    def _get_sex_selection(self):
+        return [
+            ('m', 'Male'),
+            ('f', 'Female'),
+        ]
+
+    def _get_marital_status_selection(self):
+        return [
+            ('s', 'Single'),
+            ('m', 'Married'),
+            ('w', 'Widowed'),
+            ('d', 'Divorced'),
+            ('x', 'Separated'),
+            ('z', 'law marriage'),
+        ]
+
+    @api.model
+    def _get_default_patient_id(self):
         """ Gives default patient_id """
-        patient_ids = self.search(cr, uid, [('name', '=', 'Libre')],
-                                  context=context)
-        if not patient_ids:
-            raise orm.except_orm(_('Error!'), _('No default patient defined'))
-
-        return patient_ids[0]
-
-    def _get_age(self, cr, uid, ids, field_name, arg, context=None):
-        res = {}
-        age = ''
+        domain = [('name', '=', 'Libre')]
+        patient_ids = self.search(domain)
+        if not len(patient_ids) == 1:
+            raise Warning(_('No default patient defined'))
+        return patient_ids.id
+
+    @api.one
+    @api.depends('deceased', 'dod', 'dob')
```

Comment:
I fear this wont work as this stored field will only be computed on its triggers. You might want to keep it unstored as age depends on current day.

Maybe we want an other stored field like age_on_death